pkg/handlers/generic/lifecycle/csi/nutanix-csi/handler.go

```diff
@@ -149,8 +158,8 @@ func (n *NutanixCSI) handleHelmAddonApply(
 			ClusterSelector: metav1.LabelSelector{
 				MatchLabels: map[string]string{clusterv1.ClusterNameLabel: req.Cluster.Name},
 			},
-			ReleaseNamespace: req.Cluster.Namespace,
-			ReleaseName:      fmt.Sprintf(defaultStorageHelmReleaseNameTemplate, req.Cluster.Name),
```

Comment:
The release name doesn't need the cluster-name since it will already be unique running on the remote cluster.
Only the cluster-autoscaler addon needs the cluster.Name suffix since those are always run on the single management cluster.